region/lowernorfair/east.json

```diff
@@ -6235,7 +6233,7 @@
                   "note": "Use a Morph Bomb to get up and over the Fune."
                 },
                 {
-                  "name": "Fireflea Room Fune Boost",
+                  "name": "LN Fireflea Room Damage Boost Over the Fune",
```

Comment:
Thanks, I was trying to figure that out. 
Should it stay notable since it's tighter than a 3 tile? Or could I just put 2 tile on there?
Can you escape right to left with a 2 tile mid air?

Reply:
It feels a lot like doing a 3High morph.  Trying a 2High morph is the natural approach to it which include a walljump morph.   I'm thinking you `OR` both of those.
You can't right to left without another item, or if you can its too precise for me to do.

Reply:
note: Try to hit the flame as soon as it spawns.